sys-block/arcconf/arcconf-2.01.22270-r1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV="v${PV//./_}"
+
+inherit linux-info
+
+DESCRIPTION="Microsemi Adaptec RAID Controller Command Line Utility"
+HOMEPAGE="https://www.microsemi.com/"
+SRC_URI="https://download.adaptec.com/raid/storage_manager/${PN}_${MY_PV}.zip"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="Microsemi"
+SLOT="0"
+
+DEPEND="app-arch/unzip"
+RDEPEND="${RDEPEND}"
```

Comment:
Sure `app-arch/unzip` is needed at runtime?

Reply:
Nope, that was a misstake. Removed.